doc/decisions/STEPS.md

```diff
@@ -90,9 +120,20 @@ The "Implication" must clearly say how much of the decision is already implement
 - The decision links to the new documentation.
 
 > In this step, decision PRs only modify a _single_ decision.
-> Here more exceptions are allowed, in particular documentation updates concerning the decision.
+> Here more exceptions are allowed, in particular documentation updates are okay.
+
+## Postponed
+
+This step is:
+
+- for decisions that are useful contributions but currently there is simply no urgency or immanent need for the proposal.
```

Comment:
```suggestion
- for decisions that would be useful contributions but there is currently nobody with the time to focus on the problem.
```

If there is a clear problem and there is a person who wants to work on it, why not let them? They may not want to or have the skill to work on other problems, so refusing their contribution would just be a waste.